proto/account/command.proto

```diff
@@ -68,3 +69,15 @@ message ChangeAPIKeyNameCommand {
 message EnableAPIKeyCommand {}
 
 message DisableAPIKeyCommand {}
+
+message CreateSearchFilterCommand {
+  string name = 1;
+  string query = 2;
+  account.FilterTargetType filter_target_type = 3;
+  string environment_id = 4;
```

Comment:
We don't need the environment ID in the command. It's set in the Request.

Reply:
I understand that filters are saved separately for each Env, but is this correct?
In that case, it is necessary to retain Env information, so I think it is necessary to include Env information in the command.What do you think?
Or is account information saved as a record for each Env?

Reply:
Sorry. It's correct as it is.